db/migrations/03_add_description_column.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE listings ADD COLUMN description VARCHAR(1000);
```

Comment:
Perhaps change title of Doc. There is already a 03_..._.sql file present. This should be looked at when we have merged!